ksqldb-common/src/main/java/io/confluent/ksql/util/QueryApplicationId.java

```diff
@@ -25,10 +25,35 @@
   private QueryApplicationId() {
   }
 
+  public static String buildSharedRuntimeId(
+      final KsqlConfig config,
+      final boolean persistent,
+      final int sharedRuntimeIndex
+  ) {
+    final String queryAppId = buildPrefix(config, persistent) + "-" + sharedRuntimeIndex;
+    if (persistent) {
+      return queryAppId;
+    } else {
+      return addTimeSuffix(queryAppId);
```

Comment:
@walker I changed the application id for shared runtimes very slightly, to use this "time suffix" instead of a UUID, which is what we seem to be doing for dedicated KafkaStreams applications currently and should be sufficient for getting a unique app id. This'll probably end up being removed in the next PR or two but I'm trying to keep this PR logically the same so it's easy to merge, so just swapping a UUID for a timestamp seemed like a trivial enough change for this PR

Reply:
Thats not my tag :). but yeah I don't mind that change